test/integration/network-integration.cfg

```diff
@@ -3,6 +3,7 @@
 
 [defaults]
 host_key_checking = False
+timeout = 30
```

Comment:
This file is used by the `network-integration` tests in the `ansible/ansible` repository only.

The configuration in the collection will need to be updated instead: https://github.com/ansible-collections/vyos.vyos/blob/master/tests/integration/network-integration.cfg